plugins/sndio/CMakeLists.txt

```diff
@@ -1,35 +1,20 @@
 project(sndio)
 
-if(DISABLE_SNDIO)
-	message(STATUS "Sndio support disabled")
-	return()
+option(ENABLE_SNDIO "Build OBS with sndio support" OFF)
+if(NOT ENABLE_SNDIO)
+  message(STATUS "OBS: sndio support not enabled")
+  return()
 endif()
 
-find_package(Sndio)
-if(NOT Sndio_FOUND AND ENABLE_SNDIO)
-	message(FATAL_ERROR "Sndio not found but set as enabled")
-elseif(NOT Sndio_FOUND)
-	message(STATUS "Sndio not found, disabling Sndio plugin")
-	return()
-endif()
+find_package(Sndio REQUIRED)
```

Comment:
Super minor nitpick:  everywhere else this is lowercase "sndio".  Does this need a capital "S" here?  Not a big deal either way.

Reply:
The CMake finder is called `FindSndio`, so convention requires that spelling (otherwise CMake will give you warnings and all because the names don't match).

Reply:
Got it.  Thanks for explaining.